go/interpreter/lfvm/instructions.go

```diff
@@ -1082,11 +1082,17 @@ func neededMemorySize(c *context, offset, size *uint256.Int) (uint64, error) {
 	return offset.Uint64() + size.Uint64(), nil
 }
 
-func genericCall(c *context, kind tosca.CallKind) {
-	warmAccess, coldCost, err := addressInAccessList(c)
-	if err != nil {
-		return
+func getAccessCost(accessStatus tosca.AccessStatus) tosca.Gas {
+	// EIP-2929 says that cold access cost is 2600 and warm is 100.
+	// however, the warm access cost is charged as part of the base gas cost.
+	// (https://eips.ethereum.org/EIPS/eip-2929)
+	if accessStatus == tosca.ColdAccess {
+		return tosca.Gas(2500)
 	}
+	return tosca.Gas(0)
```

Comment:
[EIP-2929](https://eips.ethereum.org/EIPS/eip-2929) actually states a warm-access cost of 100 and a cold account access cost of 2600. Please update correct this.

The reason why these prices currently work is that the warm access costs are already included in the static gas prices of call instructions [here](https://github.com/Fantom-foundation/Tosca/blob/ce72f47093f896f9ff5528315060a2cb2df80a17/go/interpreter/lfvm/gas.go#L82-L85). Those instructions actually have static gas costs of 0. This was documented in the code you removed in the `gas.go` file, and would be lost now.

A clean fix for this would be to reduce the static gas costs for CALL instructions to 0 and report the proper, EIP-2929 prices here.